lib/srv/desktop/rdp/rdpclient/src/cliprdr.rs

```diff
@@ -86,42 +87,47 @@ impl CliprdrBackend for TeleportCliprdrBackend {
             "CLIPRDR: on_remote_copy, available formats: {:?}",
             available_formats
         );
-        // always use CF_UNICODETEXT if available
-        // let mut format = available_formats
-        //     .iter()
-        //     .find(|cf| cf.id() == ClipboardFormatId::CF_UNICODETEXT);
-        // // if not fallback to CF_TEXT
-        let mut format: Option<&ClipboardFormat> = None;
-        if format.is_none() {
-            format = available_formats
-                .iter()
-                .find(|cf| cf.id() == ClipboardFormatId::CF_TEXT);
-        }
-        if let Some(format) = format.map(ClipboardFormat::id) {
+
+        let format = available_formats
+            .iter()
+            .find(|cf| {
+                cf.id() == ClipboardFormatId::CF_UNICODETEXT
+                    || cf.id() == ClipboardFormatId::CF_TEXT
+            })
+            .map(ClipboardFormat::id);
+
+        if let Some(format) = format {
             self.send("remote_copy", move |c| c.initiate_paste(format))
+        } else {
+            debug!(
+                "data was copied on the remote desktop, but no supported formats were found: {:?}",
+                available_formats
+            );
```

Comment:
What does the user see if they copied something with no supported formats. Is it this debug message?